ee/clickhouse/sql/events.py

```diff
@@ -98,6 +98,35 @@
     database=settings.CLICKHOUSE_DATABASE,
 )
 
+KAFKA_EVENTS_TABLE_JSON_SQL = lambda: EVENTS_TABLE_BASE_SQL.format(
+    table_name="kafka_json_events",
```

Comment:
Nit: WDYT about naming it `kafka_events` or `kafka_events_json`? Other tables are in form of `kafka_table`

Reply:
sure